services/ui-src/src/components/export/ExportedReportSubsection.tsx

```diff
@@ -0,0 +1,131 @@
+import { useContext } from "react";
+// components
+import { Box, Heading } from "@chakra-ui/react";
+import { Table, SpreadsheetWidget, ReportContext } from "components";
+// utils
+import { pareseFieldData, parseCustomHtml, parseFieldLabel } from "utils";
+// types
+import { FormJson, ReportPageVerbiage } from "types";
+
+export const ExportedReportSubsection = ({
+  content: { verbiage, form },
+}: ExportedReportSubsectionProps) => {
+  const { report } = useContext(ReportContext);
+  const sectionHeading = verbiage?.intro.subsection || name;
+  const isNotDynamicField =
+    form?.fields.filter((f) => f.type !== "dynamic").length !== 0;
+
+  const headRowItems = isNotDynamicField
+    ? ["Number", "Indicator", "Response"]
+    : ["Indicator", "Response"];
+
+  const fieldRowsItems = (field: any) => {
+    if (isNotDynamicField) {
+      return [
+        `<strong>${parseFieldLabel(field.props).indicator}</strong>`,
+        parseFieldLabel(field.props).label,
+        pareseFieldData(report?.fieldData[field.id]),
+      ];
+    }
+
+    return [`<strong>${field.props.label}</strong>`, pareseFieldData(field.id)];
```

Comment:
Are we doing dynamic stuff later?

![Screen Shot 2022-12-05 at 2 48 22 PM](https://user-images.githubusercontent.com/57802560/205729527-fc5f1e0c-54a2-4492-9359-25ed486ac7ec.png)
![Screen Shot 2022-12-05 at 2 48 14 PM](https://user-images.githubusercontent.com/57802560/205729548-0b657d85-0652-4b4a-aaee-ab591850b90c.png)

Reply:
note from Scott: to be worked later